### ISSUE TITLE
Add yourself to the ADOPTERS file, get an awesome t-shirt

### ISSUE THREAD

**Issue by aantn** [robusta-dev/krr#61]
Hey all, we'd like highlight some success stories from using KRR in [ADOPTERS.md](https://github.com/robusta-dev/krr/blob/main/ADOPTERS.md). Open a PR telling us how about your experience with KRR and we'll send you some KRR swag 🙂

**Comment by Rajpratik71** [comment]
> Hey all, we'd like highlight some success stories from using KRR in [ADOPTERS.md](https://github.com/robusta-dev/krr/blob/main/ADOPTERS.md). Open a PR telling us how about your experience with KRR and we'll send you some KRR swag 🙂

Hi @aantn  


I came across your offer to highlight success stories in the `ADOPTERS.md` file and receive KRR swag. I’m really excited about the opportunity and would like to participate.

Could you please confirm if this offer is still available? If so, are there any specific guidelines or deadlines I should be aware of?

Thank you for your time!


Best regards,
Pratik

**Comment by aantn** [comment]
@Rajpratik71 yep, still relevant! Just open a PR adding an entry to the adopters file according to the example already there! Feel free to speak to @pavangudiwada in the community if you want to consult.

**Comment by pavangudiwada** [comment]
Closed in favor of #433
